app/models/rateitapp/rating.rb

```diff
@@ -19,5 +19,12 @@ class Rating < ActiveRecord::Base
     validates :ratable_type, presence: true
     validates :ratable_id, presence: true
     validates :user_id, presence: true
+    validate :value_is_within_range
+
+    def value_is_within_range
+      if (value && ratable_type) && !(0..5).cover?(value)
+        errors.add(:value, "can't be outside range")
+      end
+    end
```

Comment:
Would http://apidock.com/rails/ActiveModel/Validations/HelperMethods/validates_inclusion_of make this more terse?